### PR TITLE
Chagesets Setup

### PR DESCRIPTION
**Pull request by gksander**
This PR adds `changesets` to automate our changeset/versioning/release process.

### CODE REVIEW

**Review by carlos-kelly — APPROVED** [review]
Wrong PR I commented on. I meant the Nuka one!

**Review by ryan-roemer — APPROVED** [review]
Looks good!

**Review by carlos-kelly — APPROVED** [review]

**Comment by Uzomahopeujunwa** [comment]
Looks nice

File: .changeset/config.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.0.0/schema.json",
+  "changelog": [
+    "@svitejs/changesets-changelog-github-compact",
+    {
+      "repo": "FormidableLabs/spectacle"
```
Comment (Uzomahopeujunwa) [inline .changeset/config.json:6]: Nice

File: .changeset/config.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.0.0/schema.json",
```
Comment (Uzomahopeujunwa) [inline .changeset/config.json:2]: Lovely

**Review by Uzomahopeujunwa — COMMENTED** [review]
Lovely

**Pull request merged**